test/runScript-mock-findBin.spec.js

```diff
@@ -4,22 +4,20 @@
 
 import mockFn from 'execa'
 import runScript from '../src/runScript'
+import findBinMock from '../src/findBin'
 
 jest.mock('execa')
+jest.mock('../src/findBin')
 
 // Mock findBin to return an absolute path
-jest.mock(
```

Comment:
I still don't get why this change is needed. I'm reading http://facebook.github.io/jest/docs/en/jest-object.html#jestmockmodulename-factory-options and it seems like a valid implementation. Am I missing it?

Reply:
Quite frankly I wasn't sure about the problem and it was just something I was trying to ensure consistent successful results for the tests. After investigating this a bit more, I am suspecting this was caused by the `virtual` option.

> The third argument can be used to create virtual mocks – mocks of modules that don't exist anywhere in the system